code/modules/psionics/psionic_items/psi_Larmor.dm

```diff
@@ -0,0 +1,180 @@
+	slot_flags = SLOT_HEAD
+	flags_inv = BLOCKHAIR|BLOCKHEADHAIR|FLEXIBLEMATERIAL
+	item_flags = BLOCKHAIR|BLOCKHEADHAIR|FLEXIBLEMATERIAL
+	matter = list()
+	slowdown = -0.375 //3,75% speed up!
+	armor_list = list(
+		melee = 30,
+		bullet = 25,
+		energy = 30,
+		bomb = 25,
+		bio = 100,
+		rad = 30
+	)
+	siemens_coefficient = 0.4
+	light_color = "#5B0E4F"
+	brightness_on = 8
+	on = 0
+	light_applied = 0
+	var/mob/living/carbon/human/occultist
+	var/pointgranted = 0 //Did we give you your cog?
+	var/pointremoved = 0 //Did we take you your cog?
+
+	action_button_name = "Toggle Witchlight" //reflavor this so I can make it purple to go in line with the guns - Sigma
+	light_overlay = "helmet_light" //Sadly this has to go in icons/obj/light_overlays because I can't figure out how to point it to a different one.
+										  //Currently it's located in the icons/obj/light_overlays folder, proc is at /obj/item/clothing/head/on_update_icon(mob/user) -Sigma
+
+/obj/item/clothing/head/helmet/space/occultLhood/dropped()
+	..()
+	if(!pointremoved)
+		occultist.stats.changeStat(STAT_COG, -5)
+		pointremoved = TRUE
+	spawn(2)
+	qdel(src)
+
+
+/obj/item/clothing/head/helmet/space/occultLhood/equipped(var/mob/M)
+	.=..()
+	occultist = M
+	if(!pointgranted)
+		occultist.stats.changeStat(STAT_COG, 5)
+		pointgranted = 1
+
+/obj/item/clothing/gloves/occultLgloves //We want them to not be snippable. Maybe make it some kind of rigsuit?
+	name = "psion void gloves"
+	desc = "These are thin long gloves protected by bronze plates. Ancient formulas are engraved on the fingertips, helping to weave streams of thoughts into a single powerful desire"
+	icon = 'icons/obj/psionic/occLicon.dmi'
+	item_state = "gloves"
+	icon_state = "gloves"
+	icon_override = 'icons/obj/psionic/occLmob.dmi'
+	slot_flags = SLOT_GLOVES
+	item_flags = THICKMATERIAL
+	siemens_coefficient = 1 //Insulated!
+	matter = list()
+	armor_list = list(
+		melee = 30,
+		bullet = 25,
+		energy = 30,
+		bomb = 25,
+		bio = 100,
+		rad = 30
+	)
+	item_flags = STOPPRESSUREDAMAGE|THICKMATERIAL|AIRTIGHT //make these like spacesuit so it can be a real spacesuit
+	var/mob/living/carbon/human/occultist
+	var/pointgranted = 0 //Did we give you your cog?
+	var/pointremoved = 0 //Did we take you your cog?
+
+/obj/item/clothing/gloves/occultLgloves/dropped()
+	..()
+	if(!pointremoved)
+		occultist.stats.changeStat(STAT_COG, -5)
+		pointremoved = TRUE
+	spawn(2)
+	qdel(src)
+
+/obj/item/clothing/gloves/occultLgloves/equipped(var/mob/M)
+	.=..()
+	occultist = M
+	if(!pointgranted)
+		occultist.stats.changeStat(STAT_COG, 5)
+		pointgranted = 1
+
+/obj/item/clothing/gloves/occultgloves/attackby(obj/item/W, mob/user) //Overwrite the gloves clip proc because we don't want these clipped off at all.
```

Comment:
```suggestion
/obj/item/clothing/gloves/occultLgloves/attackby(obj/item/W, mob/user) //Overwrite the gloves clip proc because we don't want these clipped off at all.
```